### PR TITLE
bugfix/SLNT-5_Fix_the_conflict_between_ESint_and_Prettier

### PR DESCRIPTION
**Pull request by MostafaFawzy7**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = {
     "plugin:prettier/recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "next/core-web-vitals",
+    "prettier",
```
Comment (MMGAbdelhay) [inline src/index.ts:14]: we can use types with interfaces now ? don't we need to adjust the `rules --> overrides`  ?https://github.com/supaapps/mysupaapps/pull/14#discussion_r1425321431

Comment (MostafaFawzy7) [reply src/index.ts:14]: Yes, extending `prettier` will handle this mission!
Please, test it from your side to make sure of this... 🙏

Comment (MMGAbdelhay) [reply src/index.ts:14]: For sure

Comment (MostafaFawzy7) [reply src/index.ts:14]: BTW, this will accept using `Type` besides `interface` but with `semi-colons` not `commas`

Comment (MMGAbdelhay) [reply src/index.ts:14]: Not sure why but i keep getting this error when i try to link the package locally with `Mysupaapps` app. i'm sure it's not related to this PR but some local issue.

Did you got this error before ?! I've revised all the dependencies versions to make sure they are compatible but idk, maybe you got something in mind.

```
npm run lint                     

> mysupaapps@0.1.0 lint
> next lint

Plugin "prettier" was conflicted between ".eslintrc.json » eslint-config-supaapps" and ".eslintrc.json » eslint-config-supaapps » plugin:prettier/recommended".

Comment (MostafaFawzy7) [reply src/index.ts:14]: Actually, didn't get this error! 🤔

Comment (SirNarsh) [reply src/index.ts:14]: @Moamen-Elgammal if you are testing locally, with `npm link` you will need to run a `npm build` first to make sure you have the latest compiled version in your `eslint-config-supaapps` repo

**Comment by SirNarsh** [comment]
closing as I duplicated this in `SLNT-6`

**Pull request closed without merging**